src/App/src/Handler/HomePageHandler.php

```diff
@@ -38,6 +44,23 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
 
         $data = [];
 
+        if ('Aura\Di\Container' == $this->containerName) {
```

Comment:
can use `===` comparison, ref https://3v4l.org/YDr7A

Reply:
I would go with `switch` in that case